README.md

```diff
@@ -85,6 +85,18 @@ $fileContent = $client->download([
 ]);
 ```
 
+#### File Copy
+``` php
+$copyOfFile = $client->copy([   
+    'BucketName' => $bucketName,
+    'FileName'   => $path,
+    'SaveAs'     => $newPath,
+
+    // Can also supply BucketId instead of BucketName
+    // Optional is destinationBucketName or destinationBucketId 
```

Comment:
Please change your `destinationBucket` parameters to be `DestinationBucket` just to keep things consistent.

Reply:
Please update the capital D for Destination in the documentation here, then squash to a single commit for me again. Thanks.